kratos/processes/find_global_nodal_neighbours_for_entities_process.cpp

```diff
@@ -237,10 +237,16 @@ template <class TContainerType>
 void FindNodalNeighboursForEntitiesProcess<TContainerType>::ClearNeighbours()
 {
     block_for_each(this->mrModelPart.Nodes(), [&](NodeType& rNode) {
-        auto& r_gp_neighbour_nodes_vector = rNode.GetValue(mrOutputVariable);
-        r_gp_neighbour_nodes_vector.erase(r_gp_neighbour_nodes_vector.begin(),
-                                          r_gp_neighbour_nodes_vector.end());
-        r_gp_neighbour_nodes_vector.shrink_to_fit();
+        if(rNode.Has(mrOutputVariable)){
+            auto& r_gp_neighbour_nodes_vector = rNode.GetValue(mrOutputVariable);
```

Comment:
is `r_gp_neighbour_nodes_vector` referenced somewhere else?
If not then probably the `else` branch would be sufficient